source/shared/net/resources/glossary.rst

```diff
@@ -233,8 +233,7 @@ An `ElGamal`_ public key associated to an account which is used to encrypt all
 Epoch
 =====
 
-A time period consisting of multiple :ref:`slots<glossary-slot>` (on the testnet, an epoch lasts
-for approximately one hour). At the start of each epoch, we compute a
+A time period consisting of multiple :ref:`slots<glossary-slot>`. An epoch is one hour on testnet and mainnet. At the start of each epoch, we compute a
```

Comment:
I'm not sure about "we compute", as it suggests it is done by people rather than an automatic process.